package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "The open source modular smart mirror platform.",
 	"main": "js/electron.js",
 	"scripts": {
-		"start": "DISPLAY=\"${DISPLAY:=:0}\" ./node_modules/.bin/electron js/electron.js",
+		"start": "./node_modules/.bin/electron js/electron.js",
```

Comment:
the `DISPLAY` var is needed (except on windows) so removing this is no good idea.

Reply:
Thank you. My poor git skills. I need to remove both the package.json and the package-lock.json files from this PR.